components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-controls/reporting-controls.component.html

```diff
@@ -2,7 +2,7 @@
   <img class="help-icon" src="/assets/img/profiles.svg" alt="">
   <div class="help-msg">
     <p>A list of of the compliance controls and their status for your scan report will appear here.</p>
-    <p>Learn how to start your first <a href="https://docs.chef.io/integrate_compliance_chef_automate.html" target="_blank">compliance scan</a>.</p>
+    <p>Learn how to start your first <a href="https://automate.chef.io/docs/data-collection/#setting-up-chef-infra-client-to-send-compliance-scan-data-through-the-chef-infra-server-to-chef-automate" target="_blank">compliance scan</a>.</p>
```

Comment:
```suggestion
    <p>Learn how to start your first <a href="https://automate.chef.io/docs/data-collection" target="_blank">compliance scan</a>.</p>
```
My thinking is that since we don't know if a user is collecting data from an Infra Server or directly from the chef-client, it makes sense to send them to the page rather than a more specific heading.